tools/travis/.travis.yml

```diff
@@ -12,12 +12,13 @@ php:
 env:
   global:
     - PATH="$HOME/.composer/vendor/bin:$PATH"
+    - SYMFONY_DEPRECATIONS_HELPER=weak
 
 matrix:
   fast_finish: true
   include:
     - php: 5.3
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest --prefer-stable"
```

Comment:
With the following `composer.json`:

```json
{
    "name": "root/composer-check",
    "require": {
        "symfony/http-kernel": "^2.3"
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

And the following command:

```bash
composer up --prefer-lowest
```

I got:

```bash
composer info 
psr/log                  1.0.0  Common interface for logging libraries
symfony/debug            v2.3.0 Symfony Debug Component
symfony/event-dispatcher v2.1.0 Symfony EventDispatcher Component
symfony/http-foundation  v2.2.0 Symfony HttpFoundation Component
symfony/http-kernel      v2.3.0 Symfony HttpKernel Component
```